sphinx_wagtail_theme/footer.html

```diff
@@ -2,7 +2,7 @@
     <div class="row">
         <div class="col p-4">
             {% if theme_footer_links %}
-                <nav>
+                <nav aria-label="Footer navigation">
```

Comment:
```suggestion
                <nav aria-label="Footer">
```

Otherwise this would be announced as `Footer navigation navigation`.